lib/vrl/stdlib/src/parse_nginx_log.rs

```diff
@@ -358,5 +361,27 @@ mod tests {
             }),
             tdef: TypeDef::object(kind_error()).fallible(),
         }
+
+        error_rate_limit {
+            args: func_args![
+                value: r#"2022/05/30 20:56:22 [error] 7164#7164: *38068741 limiting requests, excess: 50.416 by zone "api_access_token", client: 10.244.0.0, server: test.local, request: "GET / HTTP/2.0", host: "127.0.0.1:8080""#,
```

Comment:
I've read about the format [in one of Nginx blog articles](https://www.nginx.com/blog/rate-limiting-nginx/). Another option is to read the [module code](https://github.com/nginx/nginx/blob/master/src/http/modules/ngx_http_limit_req_module.c) to find the exact format, but I usually try to avoid it 😅